### PR TITLE
SAK-48113: PA System: B-5 update

### PR DESCRIPTION
**Pull request by kunaljaykam**
https://sakaiproject.atlassian.net/browse/SAK-48113

### CODE REVIEW

File: pasystem/pasystem-tool/tool/src/webapp/scripts/pasystem-tool.js [hunk 1]
```diff
@@ -29,22 +29,28 @@ $(function () {
...
+                        });
 
-        $(this).closest(".portletBody").append($modal);
+          this.closest(".portletBody").insertAdjacentHTML('beforeend', modal);
 
-        $modal.modal();
-      });
+          var modal_id = document.getElementById("pasystemDeleteConfirmationModal");
+
+          var bsModal = new bootstrap.Modal(modal_id);
```
Comment (adrianfish) [inline pasystem/pasystem-tool/tool/src/webapp/scripts/pasystem-tool.js:50]:
```suggestion
          const modalId = document.getElementById("pasystemDeleteConfirmationModal");
          (new bootstrap.Modal(modalId)).show();
```

File: pasystem/pasystem-tool/tool/src/webapp/scripts/pasystem-tool.js [hunk 2]
```diff
@@ -29,22 +29,28 @@ $(function () {
...
+          bsModal.show();
```
Comment (adrianfish) [inline pasystem/pasystem-tool/tool/src/webapp/scripts/pasystem-tool.js:51]:
```suggestion
```

**Review by adrianfish — CHANGES_REQUESTED** [review]
You've left a few typos in there, and I added a few js changes.

File: library/src/skins/default/src/sass/base/_icons.scss [hunk 3]
```diff
@@ -139,6 +139,8 @@ $fa-font-path: "./fonts";
     hidden: bi-eye-slash-fill,
     locked: bi-lock-fill,
     link : bi-link-45deg,
+    square-x-icon : bi-x-square,
+    sakai-delete-icon : bi-trash,
```
[the remark this answers is not in the document]
Comment (kunaljaykam) [reply library/src/skins/default/src/sass/base/_icons.scss:143]: You are right, We shouldn't. Thanks

**Review by adrianfish — APPROVED** [review]

**Pull request merged**